### PR TITLE
Add new functionality to get versions by providing a GAV

### PR DESCRIPTION
**Pull request by harshmadhani**
### Checklist:

* [ ] Have you added a note in the [CHANGELOG wiki](https://github.com/project-ncl/bacon/wiki/Changelog) for your change if user-facing?
* [ ] Have you added unit tests for your change?

### CODE REVIEW

File: da/src/main/java/org/jboss/bacon/da/DAReportsCli.java [hunk 1]
```diff
@@ -2,18 +2,18 @@
...
 import lombok.extern.slf4j.Slf4j;
 import org.jboss.bacon.da.rest.endpoint.ReportsApi;
+import org.jboss.da.model.rest.GAV;
 import org.jboss.da.model.rest.NPMPackage;
 import org.jboss.da.reports.model.api.SCMLocator;
-import org.jboss.da.reports.model.request.AlignReportRequest;
-import org.jboss.da.reports.model.request.BuiltReportRequest;
-import org.jboss.da.reports.model.request.SCMReportRequest;
-import org.jboss.da.reports.model.request.VersionsNPMRequest;
+import org.jboss.da.reports.model.request.*;
```
Comment (thescouser89) [inline da/src/main/java/org/jboss/bacon/da/DAReportsCli.java:8]: The PNC style kinda dicates that you don't use the catch-all '*' import. Could you please revert it back to explicitly having all the import lines one by one pelase?

**Review by thescouser89 — CHANGES_REQUESTED** [review]

File: da/src/main/java/org/jboss/bacon/da/DAReportsCli.java (hunk 1 above)
Comment (harshmadhani) [reply da/src/main/java/org/jboss/bacon/da/DAReportsCli.java]: Sure, done

File: da/src/main/java/org/jboss/bacon/da/DAReportsCli.java [hunk 2]
```diff
@@ -220,4 +221,35 @@ public Integer call() throws Exception {
             return 0;
         }
     }
+
+    @CommandLine.Command(name = "lookupGav", description = "Get built artifacts for GAVs specified.")
```
[the remark this answers is not in the document]
Comment (harshmadhani) [reply da/src/main/java/org/jboss/bacon/da/DAReportsCli.java]: Sure, done !

**Review by thescouser89 — APPROVED** [review]
LGTM! Thanks for the PR! :)

**Comment by harshmadhani** [comment]
Thanks @thescouser89  !

**Pull request merged**